### PR TITLE
Add namespace prefix to launcher, 3scale, monitoring and apicurito

### PR DESCRIPTION
**Pull request by matskiv**
## Additional Information
**WIP**

Main JIRA: https://issues.jboss.org/browse/INTLY-1400
Sub-tasks implemented here:
https://issues.jboss.org/browse/INTLY-1426
https://issues.jboss.org/browse/INTLY-1424
https://issues.jboss.org/browse/INTLY-1432
https://issues.jboss.org/browse/INTLY-1429

## Verification Steps
_TODO_

### CODE REVIEW

File: roles/middleware_monitoring/defaults/main.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
-monitoring_namespace: middleware-monitoring
+monitoring_namespace: "{{ns_prefix | default('')}}middleware-monitoring"
+monitoring_display_name: "RHMI Monitoring"
```
Comment (matskiv) [inline roles/middleware_monitoring/defaults/main.yml]: Please, suggest a display name for monitoring namespace :)

**Comment by matskiv** [comment]
I did a small test with what I had up to this point.
Installation with launcher, 3scale and monitoring finished fine. Log: https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/installation-pipeline/705/consoleFull
I could access both launcher and 3scale via a link from web app. Prometheus shows all alerts.

File: inventories/group_vars/all/manifest.yaml [hunk 2]
```diff
@@ -8,7 +8,8 @@ threescale: true
 # below threescale variables below are not currently used. They define where and what to pull in from github. These will be important as we look to break out and decouple the resources from the installer ###############
 # not currently used but records the version installed by the operator, this should come from the status block
 threescale_version: '2.4.0.GA'
-threescale_namespace: '3scale'
+threescale_namespace: "{{ns_prefix | default('')}}3scale"
```
Comment (maleck13) [inline inventories/group_vars/all/manifest.yaml]: wondering do we need one in common and one in manifest seems a duplicate

**Comment by maleck13** [comment]
👀

File: inventories/group_vars/all/manifest.yaml (hunk 2 above)
Comment (matskiv) [reply inventories/group_vars/all/manifest.yaml]: I agree. I just wanted to keep the changes to minimum.
There is more things that should be cleaned up or gently refactored. :)

**Comment by maleck13** [comment]
could you do a quick rebase

File: roles/middleware_monitoring/defaults/main.yml (hunk 1 above)
Comment (maleck13) [reply roles/middleware_monitoring/defaults/main.yml]: Managed Service Monitoring

**Comment by maleck13** [comment]
**Verification**

Install completed with ```-e 'ns_prefix=openshift-'```

**Apicutio**
Is visible from webapp and can reach it via webapp link

File: roles/3scale/templates/oauth-client.yaml.j2 [hunk 3]
```diff
@@ -1,7 +1,7 @@
 kind: OAuthClient
 apiVersion: v1
 metadata:
-  name: 3scale
+  name: "{{ threescale_oauth_client_id }}"
```
Comment (matskiv) [inline roles/3scale/templates/oauth-client.yaml.j2]: btw: I noticed we used variable in other places, but the name was hard-coded here^

**Comment by matskiv** [comment]
Squashed.

**Pull request merged**